### ISSUE TITLE
we cant comment on comments and no auto correction of spelling

### ISSUE THREAD

**Issue by Velociped** [keybase/keybase-issues#3192]
we cant comment on comments, I think it is a limitation

also there is no auto correction of spelling

**Comment by VictorKroese** [comment]
Great ideas! Having threads will keep chats better organised and more easy to find back what belongs together. Now it can be hard to follow a conversation in a channel.
Spell check/auto correct will be very nice to have.
